src/components/Web3ReactManager/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react'
+import { useWeb3React } from '@web3-react/core'
+
+import useEagerConnect from 'hooks/useEagerConnect'
+import useInactiveListener from 'hooks/useInactiveListener'
+import PageLoader from 'components/PageLoader'
+import { network } from 'connectors'
+
+export default function Web3ReactManager({ children }: { children: JSX.Element }) {
```

Comment:
This does not follow our pattern with typing or functions.

Reply:
Updated